config/routes.js

```diff
@@ -24,23 +24,25 @@ module.exports.routes = {
   //  ╚╩╝╚═╝╚═╝╩  ╩ ╩╚═╝╚═╝╚═╝
 
   // No apparent use for webpages right now...
-  '/': { view: 'pages/homepage' },
+  '/':                                    { view: 'pages/homepage' },
 
   //  ╔═╗╔═╗╦  ╔═╗╔╗╔╔╦╗╔═╗╔═╗╦╔╗╔╔╦╗╔═╗
   //  ╠═╣╠═╝║  ║╣ ║║║ ║║╠═╝║ ║║║║║ ║ ╚═╗
   //  ╩ ╩╩  ╩  ╚═╝╝╚╝═╩╝╩  ╚═╝╩╝╚╝ ╩ ╚═╝
 
-  'GET /api/v1/blobs/view': { action: 'blobs/view' },
-  'POST /api/v1/blobs/create': { action: 'blobs/create' },
+  'GET /api/v1/blobs/view':               { action: 'blobs/view' },
+  'POST /api/v1/blobs/create':            { action: 'blobs/create' },
+  'GET /api/v1/blobs/get-blob/:id':       { action: 'blobs/get-blob' },
```

Comment:
Convention would make this ` 'GET /api/v1/blobs/:id': `

Reply:
Fixed in [164b089](https://github.com/MLH-Fellowship/babel-sandbox-server/pull/12/commits/164b0894101b1f65642f0fe0989d0831fbe5ed51)